Clinical-Genomics/cgweb
# Existing mip samples

how to test: 
0. install on stage
1. in op create mip order as cust003
2. add a family 
3. add existing samples in ticket 493488
4. submit

Expected outcome:
Order successfully submitted
